mkt/reviewers/views.py

```diff
@@ -995,6 +996,69 @@ def post(self, request, pk, region, *args, **kwargs):
         return Response({'approved': bool(form.cleaned_data['approve'])})
 
 
+class _AppAction(SlugOrIdMixin):
+    permission_classes = [GroupPermission('Apps', 'Review')]
```

Comment:
AppDisable action, and AppEscalate (only when clearing/delete()) action need Apps:Edit permission. process_disable() does check this, but process_clear_escalation() doesn't currently.